partials/navbar.html

```diff
@@ -20,6 +20,12 @@
         <i class="glyphicon glyphicon-chevron-right"></i>
       </a>
     </li>
+    <li>
+      <a href="#nav-eligibility" data-ng-click="scrollTo('nav-eligibility')">
+        <div class=navlabel>Retirement Benefits Eligibility</div>
+        <i class="glyphicon glyphicon-chevron-right"></i>
+      </a>
+    </li>
     <li>
```

Comment:
We should add the isEligible check on some of the `<li>` elements in this file as well, so they are hidden when the corresponding sections are hidden.